pykrige/ok.py

```diff
@@ -632,8 +597,8 @@ def execute(self, style, xpoints, ypoints, mask=None, backend='vectorized'):
         if style != 'grid' and style != 'masked' and style != 'points':
             raise ValueError("style argument must be 'grid', 'points', or 'masked'")
 
-        xpoints = np.array(xpoints, copy=True).flatten()
-        ypoints = np.array(ypoints, copy=True).flatten()

```

Comment:
I am not sure why input arrays were flatten here before, if for instance style == 'grid'. Anyway either this could be done here or in the corresponding backend. For the moment removing it from this line, since it is handled  both by `loop` and `verctorized` backends.